### DISCUSSION TITLE
Installation frozen in 70% eval mode

### DISCUSSION THREAD

**Discussion by elsevillismo** [Security-Onion-Solutions/securityonion#9392] [General] [answered]
I can not install Security Onion in EVAL mode, installation gets frozen in 70%. 
VMware 16.5
RAM:12GB 
Cores: 4
HDD:200gb.

![Captura de pantalla 2022-12-13 220321](https://user-images.githubusercontent.com/120596628/207688252-65489282-7b5a-4e24-b2eb-12ff5d57618a.png)

**Comment by dougburks** [comment]
Did you verify the ISO image?
https://docs.securityonion.net/en/2.3/download.html

Have you tried doing a simple IMPORT installation as shown here?
https://docs.securityonion.net/en/2.3/first-time-users.html

**Reply by elsevillismo** [reply]
I solved the problem, I assigned 250Gb for installation instead 200Gb. But when I start,I run so-allow "a" option(analyst), I instroduce the IP 192.168.....in the browser but it doesn work. Maybe is caused for this error I published as yo see in the next picture?

**Reply by elsevillismo** [reply]
ISO verified. With IMPORT installation there is not  any problem,IMPORT works properly.

**Comment by elsevillismo** [comment]
I got a new error:
![Captura de pantalla 2022-12-15 185339](https://user-images.githubusercontent.com/120596628/207932863-c7e7ab6c-f1c1-4efe-8322-12d24c12d817.png)

**Comment by elsevillismo** [comment]
After the previous error, I finish the installation with this errod:
![Captura de pantalla 2022-12-15 210617](https://user-images.githubusercontent.com/120596628/207956758-58af48b2-9df6-4a09-8fab-3cdaeeda1be6.png)

**Comment by dougburks** [comment, marked as answer]
I asked these two questions previously, but I don't see answers so I'll ask again:

> Did you verify the ISO image?
> https://docs.securityonion.net/en/2.3/download.html
> 
> Have you tried doing a simple IMPORT installation as shown here?
> https://docs.securityonion.net/en/2.3/first-time-users.html

Also, for the so-preflight error, have you double-checked your networking configuration and verified that you have full Internet access?
https://docs.securityonion.net/en/2.3/firewall.html#internet-communication

**Reply by elsevillismo** [reply]
ISO is verified. There is not any problem with import installation, working properly. I have Internet access. I open the sosetup.log::

 
![Captura de pantalla 2022-12-17 001429](https://user-images.githubusercontent.com/120596628/208203842-58981a25-4eca-4e8e-b417-ea187e385c75.png)

**Reply by dougburks** [reply]
If you cannot resolve the hostname repo.securityonion.net, then that would seem to indicate a problem with your DNS configuration.

**Reply by elsevillismo** [reply]
I configure in the installation always DNS google 8.8.8.8/4.4.4.4
It´s strange because in the same network(my wifi home,same DNS) I install security onion on my latptop in the first time,eval mode withouth any problem, and it works propperly.
Maybe the problem could be with the hardware?My pc is 8 years old...
Beside, If I try to install in vmware or virtualbox with different config, for example 16gb ram and not 12gb, or 200gb and not 250GB  the installation gets frozen in 70%.
Finally when I install with this problems...I can run an execute sudo so-allow...but I open the browser and never runs:

![Captura de pantalla 2022-12-21 181029](https://user-images.githubusercontent.com/120596628/208964426-2cff5bc6-65aa-4b94-b742-a2e8a32b0d71.png)

**Reply by dougburks** [reply]
> Maybe the problem could be with the hardware?My pc is 8 years old...

If your PC is 8 years old, then it's quite possible that it doesn't meet the hardware requirements or has some other issue:
https://docs.securityonion.net/en/2.3/hardware.html

I would start with an IMPORT installation as shown at:
https://docs.securityonion.net/en/2.3/first-time-users.html

IMPORT has much lower hardware requirements so it should work much better on your older hardware. You can also enable the Analyst desktop so that you can run a browser inside the VM and not have to worry about so-allow or other networking issues:
https://docs.securityonion.net/en/2.3/analyst-vm.html#analyst-vm

**Reply by elsevillismo** [reply]
In my laptop works properly 2.3.182 but I try to update 2.3.190 and I have the same errors.

I need downgrade to 2.3.182, where is the ISO of 2.3.182??

Thanks.

**Comment by elsevillismo** [comment]
I solved the problem. I fix Vmware bridge connection to Internet, It was a problem and so It doesn´t work properly.
However I can not install 2.3.190, I only can install 2.3.182 in my PC and in my laptop.
In 2.3.190 installation gets frozen in 70%(setting up default space in Kibana) like in the first picture of this post. I don´t Know why.

Thanks.

**Reply by dougburks** [reply]
Make sure that you are doing an IMPORT installation to lower the hardware requirements:
https://docs.securityonion.net/en/2.3/analyst-vm.html#analyst-vm
https://docs.securityonion.net/en/2.3/hardware.html

Make sure that you have Internet access from your VM. If you don't have Internet access, then choose the Airgap option:
https://docs.securityonion.net/en/2.3/airgap.html

If all else fails, try a different computer with more hardware resources.
